packages/providers/src.ts/base-provider.ts

```diff
@@ -185,11 +185,19 @@ export interface Avatar {
     linkage: Array<{ type: string, content: string }>;
 }
 
+export interface ProviderOptions {
+    headers?: Record<string, string>
+}
+
 let defaultFormatter: Formatter = null;
 const MIRROR_NODE_TRANSACTIONS_ENDPOINT = '/api/v1/transactions/';
 const MIRROR_NODE_CONTRACTS_RESULTS_ENDPOINT = '/api/v1/contracts/results/';
 const MIRROR_NODE_CONTRACTS_ENDPOINT = '/api/v1/contracts/';
 
+const localConsensusNodeId = '0.0.3';
```

Comment:
These constants are used only once, so we can just hardcode them in their places